mtp_noms_ops/apps/settings/views.py

```diff
@@ -152,3 +152,38 @@ def get_context_data(self, **kwargs):
         context = super().get_context_data(**kwargs)
         context['prisons'] = self.request.user_prisons
         return context
+
+
+class JobInformationView(FormView):
+    title = _('Help us improve this service')
+    template_name = 'settings/job-information.html'
+    success_url = reverse_lazy('security:dashboard')
+    form_class = JobInformationForm
+
+    def get_success_url(self):
+        if 'next' in self.request.GET:
+            return '{path}?{query}'.format(
+                path=self.success_url,
+                query=urlencode({'next': self.request.GET['next']})
+            )
+        return self.success_url
+
+    def form_valid(self, *args, **kwargs):
+        if has_provided_job_information(self.request.user):
+            return redirect(self.get_success_url())
+
+        if self.form_class(self.request.POST).is_valid():
+            session = get_api_session(self.request)
+            if self.request.POST['job_title'] == 'Other' and self.request.POST['other_title'] != '':
+                job_title = self.request.POST['other_title']
+            else:
+                job_title = self.request.POST['job_title']
+
+            session.post('/job-information/', json={'title': job_title,
+                                                    'prison_estate': self.request.POST['prison_estate'],
+                                                    'tasks': self.request.POST['tasks']})
+
+            save_user_flags(self.request, provided_job_info_flag)
+            return redirect(self.get_success_url())
+        else:
+            self.form_class(self.request.POST)
```

Comment:
this line has no effect

Reply:
Yep, this is another hangover from initially working with a `post` method